artifacts/attributes.adoc

```diff
@@ -67,3 +67,24 @@
 
 :LinkPluginsGuide: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/configuring_plugins_in_red_hat_developer_hub/index
 :NameOfPluginsGuide: Configuring plugins in {product}
+
+:authentication-book-url: https://docs.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/authentication/index
```

Comment:
Where does `/us-en/` come from?

https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/authentication/index works as does https://docs.redhat.com/us-en/documentation/red_hat_developer_hub/1.2/html/authentication/index and https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html-single/authentication/index


https://docs.redhat.com/us-en/documentation/red_hat_developer_hub/1.2/html-single/authentication/index returns a 404.

I wonder if removing `/us-en/` would resolve the issue. `/us-en/` is not present in any of the other URLs in this PR.

Reply:
@themr0c ^